rainier-core/src/main/scala/com/stripe/rainier/compute/Real.scala

```diff
@@ -104,12 +104,12 @@ Because it is common for ax to have a large number of terms, this is deliberatel
 as equality comparisons would be too expensive. The impact of this is subtle, see [0] at the bottom of this file
 for an example.
  */
-private final class Line private (val ax: Map[NonConstant, BigDecimal],
+private final class Line private (val ax: ListMap[NonConstant, BigDecimal],
```

Comment:
Along with the change to ListMap, I think we should change Line to a case class (so that it uses structural equality instead of identity equality). Incidentally I checked the equality semantics on ListMap and they're the same as Map, ie order isn't considered - which I think is good, for our purposes.

Reply:
We can leave this for the capped size listmap change.